popSimManu.tex

```diff
@@ -528,57 +528,92 @@ \subsection*{Model QC procedures}
 \href{https://stdpopsim.readthedocs.io/en/latest/development.html#}{developer documentation}.
 
 \subsection*{Workflow for analysis of simulated data}
-Snakemake pipeline for 1 pop models
+To demonstrate the utility of \stdpopsim we created \texttt{Snakemake}
+workflows \citep{koster2012snakemake} that perform demographic inference on
+tree sequence output from our package using a few common software packages. 
+Our choice of \texttt{Snakemake} allows complete reproducibility of the 
+analyses shown and the entire Analysis repository is available from
+\url{https://github.com/popgensims/analysis}. 
+
+We performed demographic analysis for one and two population tasks.
+Our one population task was to infer historical population size over
+time ($N(t)$). This was done using three software packages \stairwayplot
+which uses site frequency spectrum information only \citep{liu2015exploring},
+the SMC based \MSMC~\citep{schiffels2014inferring} which we ran with two different
+sample sizes $n\in (2 , 8)$, and \smcpp~\citep{terhorst2017robust} 
+which combines information from the site frequency spectrum with 
+recombination information as in SMC-based methods. No attempt
+was made at trying to optimize the analysis from
+any particular software package in present case,
+as our goal was not to benchmark performance of methods but
+instead show how such benchmarking could be easily done using
+the \stdpopsim resource. In this spirit we ran each software package as near
+to default parameters as possibile in each case. For \stairwayplot we
+set the parameters numRuns=1 and dimFactor=5000. For \smcpp we used the
+``estimate" run mode to infer $N(t)$ with all other parameters set
+to their default values. For \MSMC we used the ``--fixedRecombination"
+option and used the default number of iterations.
+
+For the single population task we ran human, HomSap, simulations
+using a variety of models: OutOfAfricaArchaicAdmixture\_5R19, OutOfAfrica\_3G09,
+a constant sized generic model, and a two epoch generic model where the
+population size instantaneously decreases from $N=10^4$ to $N=10^3$ 500 
+generations before the present. Each HomSap simulation was run
+using the HapmapII\_GRCh37 genetic maps. For \emph{D. melanogaster}
+we estimated $N(t)$ from an African sample simulated under the DroMel, 
+African3Epoch\_1S16 model using the Comeron2012\_dm6 map. Finally we ran
+simulations of \emph{A. thaliana} genomes using the AraTha African2Epoch\_1H18
+model under the Salome2012\_TAIR7. For each model, 3 replicate whole genomes were
```

Comment:
should this read "under the Salome2012 genetic map"?

Reply:
this is the new naming scheme that we are moving the paper to. see https://github.com/popgensims/manuscript/issues/42